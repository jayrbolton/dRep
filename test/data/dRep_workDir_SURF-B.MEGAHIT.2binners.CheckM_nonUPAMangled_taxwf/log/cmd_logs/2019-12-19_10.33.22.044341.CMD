prodigal -i /home/sumin/kbsdk-workspace/dRep/test/data/binned_contig_files_8bins/Bin.039.fasta -d /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/prodigal/Bin.039.fasta.fna -a /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/prodigal/Bin.039.fasta.faa -m -p meta
