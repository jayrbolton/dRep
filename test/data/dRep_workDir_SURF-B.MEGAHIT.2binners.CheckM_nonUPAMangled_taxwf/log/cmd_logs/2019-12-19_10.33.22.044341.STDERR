-------------------------------------
PRODIGAL v2.6.3 [February, 2016]         
Univ of Tenn / Oak Ridge National Lab
Doug Hyatt, Loren Hauser, et al.     
-------------------------------------
Request:  Metagenomic, Phase:  Training
Initializing training files...done!
-------------------------------------
Request:  Metagenomic, Phase:  Gene Finding
Finding genes in sequence #1 (51408 bp)...done!
Finding genes in sequence #2 (46144 bp)...done!
Finding genes in sequence #3 (168360 bp)...done!
Finding genes in sequence #4 (36840 bp)...done!
Finding genes in sequence #5 (98122 bp)...done!
Finding genes in sequence #6 (38403 bp)...done!
Finding genes in sequence #7 (5550 bp)...done!
Finding genes in sequence #8 (50923 bp)...done!
Finding genes in sequence #9 (42893 bp)...done!
Finding genes in sequence #10 (51754 bp)...done!
Finding genes in sequence #11 (7556 bp)...done!
Finding genes in sequence #12 (57096 bp)...done!
Finding genes in sequence #13 (48728 bp)...done!
Finding genes in sequence #14 (19523 bp)...done!
Finding genes in sequence #15 (73065 bp)...done!
Finding genes in sequence #16 (32190 bp)...done!
Finding genes in sequence #17 (52268 bp)...done!
Finding genes in sequence #18 (107486 bp)...done!
Finding genes in sequence #19 (13522 bp)...done!
Finding genes in sequence #20 (10843 bp)...done!
Finding genes in sequence #21 (35480 bp)...done!
Finding genes in sequence #22 (34029 bp)...done!
Finding genes in sequence #23 (49332 bp)...done!
Finding genes in sequence #24 (24519 bp)...done!
Finding genes in sequence #25 (40668 bp)...done!
Finding genes in sequence #26 (6447 bp)...done!
Finding genes in sequence #27 (20419 bp)...done!
Finding genes in sequence #28 (103403 bp)...done!
Finding genes in sequence #29 (37767 bp)...done!
Finding genes in sequence #30 (28353 bp)...done!
Finding genes in sequence #31 (30012 bp)...done!
Finding genes in sequence #32 (26830 bp)...done!
Finding genes in sequence #33 (45608 bp)...done!
Finding genes in sequence #34 (39847 bp)...done!
Finding genes in sequence #35 (8995 bp)...done!
Finding genes in sequence #36 (20377 bp)...done!
Finding genes in sequence #37 (37439 bp)...done!
Finding genes in sequence #38 (13715 bp)...done!
Finding genes in sequence #39 (23179 bp)...done!
Finding genes in sequence #40 (3598 bp)...done!
Finding genes in sequence #41 (44502 bp)...done!
Finding genes in sequence #42 (50672 bp)...done!
Finding genes in sequence #43 (51876 bp)...done!
Finding genes in sequence #44 (9729 bp)...done!
Finding genes in sequence #45 (8373 bp)...done!
Finding genes in sequence #46 (147991 bp)...done!
Finding genes in sequence #47 (35801 bp)...done!
Finding genes in sequence #48 (49751 bp)...done!
Finding genes in sequence #49 (13801 bp)...done!
Finding genes in sequence #50 (25870 bp)...done!
Finding genes in sequence #51 (116755 bp)...done!
Finding genes in sequence #52 (34423 bp)...done!
Finding genes in sequence #53 (49590 bp)...done!
Finding genes in sequence #54 (45394 bp)...done!
Finding genes in sequence #55 (31834 bp)...done!
Finding genes in sequence #56 (17587 bp)...done!
Finding genes in sequence #57 (2884 bp)...done!
Finding genes in sequence #58 (41518 bp)...done!
Finding genes in sequence #59 (53757 bp)...done!
Finding genes in sequence #60 (16975 bp)...done!
Finding genes in sequence #61 (6388 bp)...done!
Finding genes in sequence #62 (14387 bp)...done!
Finding genes in sequence #63 (40997 bp)...done!
Finding genes in sequence #64 (55539 bp)...done!
Finding genes in sequence #65 (5101 bp)...done!
Finding genes in sequence #66 (14952 bp)...done!
Finding genes in sequence #67 (8882 bp)...done!
Finding genes in sequence #68 (76030 bp)...done!
Finding genes in sequence #69 (5548 bp)...done!
Finding genes in sequence #70 (88713 bp)...done!
Finding genes in sequence #71 (182874 bp)...done!
Finding genes in sequence #72 (52818 bp)...done!
Finding genes in sequence #73 (89519 bp)...done!
Finding genes in sequence #74 (40635 bp)...done!
Finding genes in sequence #75 (7817 bp)...done!
Finding genes in sequence #76 (39731 bp)...done!
Finding genes in sequence #77 (21439 bp)...done!
Finding genes in sequence #78 (142821 bp)...done!
Finding genes in sequence #79 (2975 bp)...done!
Finding genes in sequence #80 (65965 bp)...done!
Finding genes in sequence #81 (42390 bp)...done!
Finding genes in sequence #82 (42971 bp)...done!
Finding genes in sequence #83 (29265 bp)...done!
Finding genes in sequence #84 (91633 bp)...done!
Finding genes in sequence #85 (8330 bp)...done!
Finding genes in sequence #86 (60654 bp)...done!
Finding genes in sequence #87 (67662 bp)...done!
Finding genes in sequence #88 (9955 bp)...done!
Finding genes in sequence #89 (9777 bp)...done!
Finding genes in sequence #90 (30644 bp)...done!
